src/setup/rgw.py

```diff
@@ -415,13 +415,13 @@ def _update_rgw_config_with_endpoints(conf: MappedConf, endpoints: dict, instanc
             Conf.set(Rgw._rgw_conf_idx, f'client.rgw-{instance}>{ep_value}', endpoints[key.name])
         Conf.set(Rgw._rgw_conf_idx, f'client.rgw-{instance}>log file', service_instance_log_file)
         # For each instance increase port value by 1.
-        # for eg. for 1st instance. port=8000
-        # for 2nd instance port=8000 + 1
+        # for eg. for 1st instance. port=22751
+        # for 2nd instance port=22751 + 1
         # port = <port> + (instance - 1)
         # TODO: read port value from endpoint url define in cluster.conf
-        port = 8000
+        port = 22751
```

Comment:
can we move these ports to const.py file as well.. tomorrow it may get changed so better to keep it as constants

Reply:
yes. agree.. we can skip this comment.